### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by adarachel**
1. Create a website consisting of 3 pages: Home, Calculator, and Quote.
2. Add your own styles to improve the look and feel. Use React basic styling (inline styles, importing CSS and pre-processed CSS files).
3. For every page, you should create a <Route>.
4. Make sure that navigation links for all 3 routes are displayed on each page.
5. Use React Router to make the routes work.

### CODE REVIEW

File: src/navbar.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+.navbar a {
+  color: #fff;
+  text-decoration: none;
+  padding: 0.5rem;
+  transition: color 0.3s ease;
+}
+
+.navbar a:hover {
+  color: #ffca28;
+}
```
Comment (cynthiainga) [inline src/navbar.css]: - [ ] Please as requested in the project requirements the layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) and you are allowed to improve the design like the background image the color but the structure should be the same 🙃 🙃. Kindly take a look at the picture below 👇 

## Home page
> |     | Your design: Home page                           | Original design: Home page                     |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | Screenshot 2023-04-14 at 17 28 44 | math_magicians_home |

## Calculator page
> |     | Your design: Calculator page                           | Original design: Calculator page                    |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | Screenshot 2023-04-14 at 17 32 25 | math_magicians_calculator |

## Quote page
> |     | Your design: Quote page                           | Original design: Quote page                     |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | Screenshot 2023-04-14 at 17 32 36 | math_magicians_quote |

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @adarachel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] React router is used correctly ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
-      <Quotes />
-      <Calculator />
-    </div>
-  );
-}
+const App = () => (
+  <Router>
+    <nav className="navbar">
+      <h1>Awesome Books</h1>
+      <ul className="navbar-links">
```
Comment (emmiiorji) [inline src/App.js:14]: [Optional] I suggest that you use a title that matches the caption of the project.

image

**Review by emmiiorji — APPROVED** [review]
Hi @adarachel, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**